src/utils/index.ts

```diff
@@ -29,11 +33,51 @@ export const balenaApi = new PinejsClientRequest(
 );
 export const apiKey = process.env.VPN_SERVICE_API_KEY;
 
-const consoleTransport = new winston.transports.Console({
-	format: winston.format.simple(),
-});
-export const logger = winston.createLogger({
-	transports: [consoleTransport],
-	exceptionHandlers: [consoleTransport],
-	exitOnError: false,
-});
+export const getLogger = (service: string, workerId?: string | number) => {
+	let workerLabel = 'master';
+	if (workerId != null) {
+		workerLabel = `worker-${workerId}`;
+	}
+	const transport = new winston.transports.Console({
+		format: winston.format.combine(
+			winston.format.colorize(),
+			winston.format.label({ label: workerLabel, message: true }),
+			winston.format.label({ label: service, message: true }),
+			winston.format.simple(),
+		),
+		level: 'debug',
+	});
+	return winston.createLogger({
+		transports: [transport],
+		exceptionHandlers: [transport],
+		exitOnError: false,
+		levels: winston.config.syslog.levels,
+	});
+};
+
+export const metricsServer = () => {
+	const app = express();
+	app.disable('x-powered-by');
+	app.get('/ping', (_req, res) => res.send('OK'));
+	app.get('/cluster_metrics', metrics.aggregateRequestHandler());
+	return app;
+};
+
+export const spawnChildren = (n: number, logger: winston.Logger) => {
+	logger.info(`spawning ${n} workers`);
+	_.times(n, i => {
+		const workerId = i + 1;
+		const restartWorker = (code?: number, signal?: string) => {
+			if (signal != null) {
+				logger.crit(`worker-${workerId} killed with signal ${signal}`);
+			}
+			if (code != null) {
+				logger.crit(`worker-${workerId} exited with code ${code}`);
+			}
+			const env = {};
+			_.merge(env, process.env, { WORKER_ID: workerId });
```

Comment:
I'm assuming you actually want `_.assign` behaviour here rather than `_.merge`, in which case you can do
```suggestion
			const env = {
				...process.env,
				WORKER_ID: workerId
			};
```